components/core/tools/scripts/lib_install/macos/install-all.sh

```diff
@@ -7,9 +7,15 @@ set -e
 set -u
 
 brew update
+
+# Install CMake v3.31.6 since we're using yaml-cpp that doesn't yet support CMake v4 yet
+cmake_formula_path=/tmp/cmake.rb
+curl --fail --show-error --location --output "${cmake_formula_path}" \
+  https://raw.githubusercontent.com/Homebrew/homebrew-core/b4e46db74e74a8c1650b38b1da222284ce1ec5ce/Formula/c/cmake.rb
+brew install --formula "${cmake_formula_path}"
```

Comment:
Can the `--formula` argument be omitted? i.e.,

```suggestion
brew install "${cmake_formula_path}"
```

Reply:
If we omit `--formula`, then `brew` will print a warning (and it's probably better to be explicit?).

Reply:
Right, i wasn't aware of the warning. Being explicit is also better. Please ignore the original suggestion.